<ui version="4.0" >
 <class>DlgBpmSchemeDlg</class>
 <widget class="QDialog" name="DlgBpmSchemeDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BPM Scheme</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>130</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSchemeName" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>10</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Scheme Name:</string>
   </property>
   <property name="buddy" >
    <cstring>txtSchemeName</cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>BPM Range</string>
   </property>
   <widget class="QSpinBox" name="spinBpmMin" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>220</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBpmMax" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
    <property name="maximum" >
     <number>220</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>20</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Max</string>
    </property>
    <property name="buddy" >
     <cstring>spinBpmMax</cstring>
       </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>20</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Min</string>
    </property>
    <property name="buddy" >
     <cstring>spinBpmMin</cstring>
       </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chkAnalyzeEntireSong" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>100</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Analyze Entire Song</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgBpmSchemeDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgBpmSchemeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
